cogs/DeepL.py

```diff
@@ -1,63 +1,69 @@
-from deepl import translate
+from typing import Optional
+import asyncio
+from concurrent.futures import ThreadPoolExecutor
+
+import deepl
 from discord import option
 from discord.ext import commands
 
 from util.EmbedBuilder import EmbedBuilder
 from util.Logging import log
 
-
-def get_language_list() -> list[str]:
-    languages = [
-        "Bulgarian",
-        "Chinese",
-        "Czech",
-        "Danish",
-        "Dutch",
-        "English",
-        "Estonian",
-        "Finnish",
-        "French",
-        "German",
-        "Greek",
-        "Hungarian",
-        "Italian",
-        "Japanese",
-        "Latvian",
-        "Lithuanian",
-        "Polish",
-        "Portuguese",
-        "Romanian",
-        "Russian",
-        "Slovak",
-        "Slovenian",
-        "Spanish",
-        "Swedish",
-    ]
-    return languages
-
-
-async def translation(
-    text, source_language, target_language, formality_tone=None
-) -> str:
-
-    formality = ["formal", "informal"]
-    languages = get_language_list()
-
-    if source_language not in languages or target_language not in languages:
+LANGUAGES = [
+    "Bulgarian",
+    "Chinese",
+    "Czech",
+    "Danish",
+    "Dutch",
+    "English",
+    "Estonian",
+    "Finnish",
+    "French",
+    "German",
+    "Greek",
+    "Hungarian",
+    "Italian",
+    "Japanese",
+    "Latvian",
+    "Lithuanian",
+    "Polish",
+    "Portuguese",
+    "Romanian",
+    "Russian",
+    "Slovak",
+    "Slovenian",
+    "Spanish",
+    "Swedish",
+]
+FORMALITY_TONES = ["Formal", "Informal"]
+
+
+async def translate(text: str, source_language: str, target_language: str, formality_tone: Optional[str] = None) -> str:
```

Comment:
def `x(c: str or None = None)` is preferred over importing the typing module for one optional typehint